### ISSUE TITLE
RFC: Component State - New service in package @ngrx/component

### ISSUE THREAD

**Issue by BioPhoton** [ngrx/platform#2187]
# Reactive Component State

Here my research on ephemeral-state-management.

- [💾 Reactive Ephemeral State](https://dev.to/rxjs/research-on-reactive-ephemeral-state-in-component-oriented-frameworks-38lk) In: Dev.to, Michael Hladky
- [💾 Design Doc - State Service](https://hackmd.io/wVkWRc3XQWmtM6YcktRTrA) In: HackMd, Michael Hladky.
-  [📦 NPM package - ngx-rx/rxjs-state](https://www.npmjs.com/package/ngx-rx-state)

Related to: 
- #858 
- #2052
- #2489

**Comment by BioPhoton** [comment]
The minimal version is included in this PR: https://github.com/ngrx/platform/pull/2046

**Comment by BioPhoton** [comment]
Closed as it was not interesting to consider

**Comment by maxime1992** [comment]
@BioPhoton what do you mean by "not interesting to consider" :scream:?

**Comment by BioPhoton** [comment]
I'm not sure why I had to move out of the code and proposals.
I moved all the code to https://github.com/BioPhoton/rx-angular and released the service under: https://www.npmjs.com/package/@rx-angular/state

But there was https://github.com/ngrx/platform/issues/2489 which is already accepted and only the naming of the API is left (as far as I could follow).

It was communicated that the release will also be in about 2 weeks.  (info from online presentation of @ngrx/component-store ).
